src/main/java/friday/logic/Logic.java

```diff
@@ -31,8 +31,8 @@ public interface Logic {
      */
     ReadOnlyFriday getAddressBook();
 
-    /** Returns an unmodifiable view of the filtered list of persons */
-    ObservableList<Student> getFilteredPersonList();
+    /** Returns an unmodifiable view of the filtered or sorted list of students */
+    ObservableList<Student> getStudentList();
```

Comment:
Good job refactoring here! Might be clearer to change method name to getShortlistedStudentList() to indicate that the list is like a filtered or sorted list.